### PR TITLE
fix(client): incorrect confirmation date (ARTP-907)

### PR DESCRIPTION
**Pull request by zhouyuhe**
Change the tradeDate to the correct date: confirmation date

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/NotificationContainer.tsx [hunk 1]
```diff
@@ -45,7 +47,7 @@ export default class NotificationContainer extends PureComponent<Props> {
       const { dealtCurrency, tradeId } = lastTradeExecutionStatus.trade
       const { terms } = currencyPair
       if (lastTradeExecutionStatus.trade.status === TradeStatus.Done) {
-        const { direction, notional, spotRate, tradeDate } = lastTradeExecutionStatus.trade
+        const { direction, notional, spotRate } = lastTradeExecutionStatus.trade
```
Comment (rikoe) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/NotificationContainer.tsx]: Can't we just pull valueDate out of the last trade execution status here, like was done with tradeDate before? I think (though I may be wrong), that valueDate and tradeDate should both be present.

**Comment by zhouyuhe** [comment]
> The PR should be made from the `release/v1.2` branch, and into that branch, not `master`, please.

ok, I will change it

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/NotificationContainer.tsx (hunk 1 above)
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/NotificationContainer.tsx]: cool, let me check

**Comment by rikoe** [comment]
Closed in favour of #1340

**Pull request closed without merging**